.github/workflows/artifact-redirector.yml

```diff
@@ -1,5 +1,9 @@
 name: CircleCI artifacts redirector
 on: [status]
+
+permissions:
+  statuses: write
```

Comment:
When reading the YAML file, it's not obvious to me that this is related to the use of `secretes.GITHUB_TOKEN`. Maybe we could add an explicit inline comment:

```suggestion
# Restrict the permissions granted to the use of secrets.GITHUB_TOKEN in this
# github actions workflow:
# https://docs.github.com/en/actions/security-guides/automatic-token-authentication
permissions:
  statuses: write
```